<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string>
    QWidget {
        background-color: #E3F0FE;
        background-position: center;
        background-repeat: no-repeat;
        background-size: cover;
    }

    QLabel {
        font-size: 14px;
        font-weight: bold;
        color: #333;
    }

    QLineEdit {
        border: 2px solid #ccc;
        border-radius: 8px;
        padding: 5px;
        font-size: 14px;
        background-color: #fff;
        selection-background-color: #7ec8e3;
    }

    QPushButton {
        background-color: #0078D7;
        color: white;
        border-radius: 8px;
        padding: 8px;
        font-size: 14px;
    }

    QPushButton:hover {
        background-color: #005a9e;
    }

    QPushButton:pressed {
        background-color: #004b87;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>100</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>BIENVENUE ! Insérer votre données personnel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>41</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>mot de passe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="motdpLineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Se connecter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mdpOublieButton">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>MDP oublié</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
